sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPage.java

```diff
@@ -44,4 +55,59 @@
+  public static final By IDENTIFICATION_SOURCE_DETAILS_COMBOBOX_INPUT =
+      By.cssSelector("#contactIdentificationSource input");
+  public static final By PROHIBITION_TO_WORK_OPTIONS = By.cssSelector("#prohibitionToWork label");
+  public static final By HOME_BASED_QUARANTINE_OPTIONS =
+      By.cssSelector("#quarantineHomePossible label");
+  public static final By QUARANTINE_COMBOBOX =
+      By.cssSelector(".v-verticallayout [location='quarantine'] [role='combobox'] div");
+  public static final By QUARANTINE_COMBOBOX_INPUT = By.cssSelector("#quarantine input");
+  public static final By HIGH_PRIORITY_CHECKBOX = By.cssSelector("#highPriority > input");
+  public static final By HIGH_PRIORITY_LABEL = By.cssSelector("#highPriority > label");
+  public static final By DIABETES_OPTIONS = By.cssSelector("#diabetes label");
+  public static final By HIV_OPTIONS = By.cssSelector("#immunodeficiencyIncludingHiv label");
+  public static final By LIVER_OPTIONS = By.cssSelector("#chronicLiverDisease label");
+  public static final By MALIGNANCY_OPTIONS = By.cssSelector("#malignancyChemotherapy label");
+  public static final By PULMONARY_OPTIONS = By.cssSelector("#chronicPulmonaryDisease label");
+  public static final By RENAL_OPTIONS = By.cssSelector("#chronicKidneyDisease label");
+  public static final By NEUROLOGIC_OPTIONS = By.cssSelector("#chronicNeurologicCondition label");
+  public static final By CARDIOVASCULAR_OPTIONS =
+      By.cssSelector("#cardiovascularDiseaseIncludingHypertension label");
+  public static final By ADDITIONAL_RELEVANT_PRE_CONDITIONS_TEXT = By.id("otherConditions");
+  public static final By VACCINATION_STATUS_COMBOBOX =
+      By.cssSelector(".v-verticallayout [location='vaccinationInfo'] [role='combobox'] div");
+  public static final By VACCINATION_STATUS_COMBOBOX_INPUT =
+      By.cssSelector("#vaccinationInfo input");
+  public static final By IMMUNOSUPPRESSIVE_THERAPY_OPTIONS =
+      By.cssSelector("#immunosuppressiveTherapyBasicDisease label");
+  public static final By CARE_OVER_60_OPTIONS = By.cssSelector("#careForPeopleOver60 label");
+  public static final By CANCEL_FOLLOW_UP_BUTTON = By.id("contactCancelFollowUp");
+  public static final By OVERWRITE_FOLLOW_UP_CHECKBOX =
+      By.cssSelector("#overwriteFollowUpUntil > input");
+  public static final By OVERWRITE_FOLLOW_UP_LABEL =
+      By.cssSelector("#overwriteFollowUpUntil > label");
+  public static final By FOLLOW_UP_UNTIL_DATE = By.cssSelector("#followUpUntil input");
+  public static final By FOLLOW_UP_STATUS_TEXT = By.id("followUpComment");
+  public static final By RESPONSIBLE_STATUS_OFFICER_COMBOBOX =
+      By.cssSelector(".v-verticallayout [location='contactOfficer'] [role='combobox'] div");
+  public static final By RESPONSIBLE_STATUS_OFFICER_COMBOBOX_INPUT =
+      By.cssSelector("#contactOfficer input");
+  public static final By GENERAL_COMMENT = By.id("additionalDetails");
```

Comment:
Please add textarea

Reply:
added _TEXT to keep the pattern in that class. resolved, imho.